cypress/support/util.js

```diff
@@ -50,3 +50,53 @@ export function getBlocksRecursive( blocks ) {
 	return allBlocks
 }
 
+/**
+ * Function for comparing two versions.
+ *
+ * @param {string} versionA
+ * @param {string} versionB
+ * @param {string} operator
+ */
+export function compareVersions( versionA, versionB, operator ) {
+	const arrA = Array( 'RC', 'alpha', 'beta' ).some( build => versionA.includes( build ) )
+		? versionA.split( '-' )[ 0 ].split( '.' ).map( num => parseInt( num ) )
+		: versionA.split( '.' ).map( num => parseInt( num ) )
+	const arrB = versionB.split( '.' ).map( num => parseInt( num ) )
+
+	if ( arrA.length === 2 ) {
+		// Append 0 to the version number.
+		arrA.push( 0 )
+	}
+
+	if ( operator === '<' ) {
+		if ( arrA[ 0 ] < arrB[ 0 ] ) {
+			return true
+		}
+		if ( arrA[ 0 ] === arrB[ 0 ] && arrA[ 1 ] < arrB[ 1 ] ) {
+			return true
+		}
+		if ( arrA.length === 3 ) {
```

Comment:
Since you're pushing `0` if the length of `arrA` is 2, This is always true right?